src/redux/books/bookSlice.js

```diff
@@ -1,42 +1,109 @@
-import { createSlice } from '@reduxjs/toolkit';
+/* eslint-disable import/no-extraneous-dependencies */
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
```

Comment:
- [ ] Please avoid disabling linters as they are very useful to keep a consistent code format and ensure the use of best practices. Instead, please consider solving the errors! 🤓 🔧